README.md

```diff
@@ -76,72 +72,89 @@ Installation
      After that, you must reboot your system to be sure everything works
      properly (libvirt iptables rules, ip forwarding, etc.).
 
-Running
--------
+## Running
 
-For a complete list of tasks, run:
+This is the workflow used to run integration tests:
 
-    $ rake -T
+* Create an ISO image to test. This ISO will contain the latest YaST packages,
+  although it's also possible to use any other image.
+* Install the system using the generated ISO and run the tests on the resulting
+  system. You can run the tests as many times as you want using the same
+  installed system.
 
-To run the testsuite, use the `test` Rake task:
+Actually, the second step can be seen as two different steps (building the
+system and running the tests). But for now, we'll keep them tied.
 
-    $ rake test
+After this brief introduction, let's go deeper into each step.
 
-This runs all tests defined in `test` directory (e.g. `test/tftp.rb`):
-* Building a KVM image by using the AutoYaST configuration file (e.g. `tftp.xml`)
-  You can watch the installation by using `virt-manager`. The image is `autoyast`.
-* Starting the built image.
-  You can watch it by using `virt-manager`. The image is `vagrant_autoyast_vm`.
-* Running RSpec tests on this machine which are defined in e.g. `test/tftp.rb`.
+### Generating an ISO
 
-To run only one single test use:
+Generating a new ISO to use in tests is as easy as typing:
 
-    $ rake test[<path_to_test_file>]
+    $ rake build_iso[sles12-sp1]
 
-e.g. `rake test[test/sles12.rb]`
+where `sles12-sp1` could be any of the values defined in
+[config/definitions.yml](https://github.com/yast/autoyast-integration-test/blob/master/config/definitions.yml).
```

Comment:
I was checking whether we could use something like http://......./LATEST but unfortunately we can't as these are ONLY for unpacked media in SLP directory.

There are two problems:
1. config/definitions.yml needs to be maintained
2. dist.suse.de needs to be accessible, so maybe you could describe where to put an ISO if you already have it and how to name it?

This is just an idea, no real use-case as we (Yast team) actually HAVE access to dist.suse.de.